lumapps/api/helpers/medias.py

```diff
@@ -0,0 +1,177 @@
+import requests
+
+
+def create_new_media(
+    client, file_data_or_path, doc_path, filename, mimetype, is_shared, lang="en"
+):
+    """ Upload a file and create a new media in LumApps media library.
+
+    Arguments:
+        client (ApiClient): The ApiClient used to make requests to the LumApps Api.
+        file_data_or_path (Union[bytes, str]): The filepath (str) or the data (bytes) to upload.
+        doc_path (str): The doc path of the media to upload, this will decide where the media will go in your media library
+                        (eg: provider=<my_provider>/site=<my_site_id>/resource=<my_parent_folder_id>)
+        filename (str): The name of the file to upload. Once uploaded the file will appear with that name.
+        mimetype (str): The mimeType fo the file to upload.
+        is_shared (bool): Wether the file is shared or not. Non shared files will only be visible by you.
+
+    Keyword Arguments:
+        lang (str): The lang of the file to upload (default: "en").
+
+    Raises:
+        Exception: The data or file path type provided is not supported.
+
+    Returns:
+        Union[dict, None]: Return the uploaded media if successfull, None otherwise.
+    """
+
+    if isinstance(file_data_or_path, str):
+        file_data = open(file_data_or_path, "rb")
```

Comment:
What about pathlib support?

Reply:
No python2 compatible, surely when we gave up on python2 suport !